README.md

```diff
@@ -81,4 +81,32 @@ func main() {
 
 ---
 
-If you want to contribute to Minimemcached, feel free to create an issue or pull request!
+## Benchmarks
+
+- Running same test cases on memcached server on a docker and minimemcached, minimemcached outperformed memcached running on docker container.
```

Comment:
Please add memcached server environment, docker image, etc as well.

It would be nice to give information to reproduce this benchmark results to user.